packages/blockchain-link-utils/src/tests/fixtures/solana.ts

```diff
@@ -0,0 +1,82 @@
+import { PublicKey } from '@solana/web3.js';
+
+const sampleMintToDetailMap = {
+    'So11111111111111111111111111111111111111112': {
+        name: 'Wrapped SOL',
+        symbol: 'WSOL',
+    },
+    'Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB': {
+        name: 'Tether',
+        symbol: 'USDT',
+    },
+    '4k3Dyjzvzp8eMZWUXbBCjEvwSkkk59S5iCNLY3QrkX6R': {
+        name: 'Raydium',
+        symbol: 'RAY',
+    },
+};
```

Comment:
Prettier complains about the strings being converted to static keys, i.e. `So11... : { ... }` instead of `'So11...': { ... }`, keeping this as is for now until we decide on how to handle these.